sklearn/utils/_estimator_html_repr.py

```diff
@@ -98,13 +98,24 @@ def _get_visual_block(estimator):
 
     # check if estimator looks like a meta estimator wraps estimators
     if hasattr(estimator, "get_params"):
-        estimators = []
-        for key, value in estimator.get_params().items():
-            # Only look at the estimators in the first layer
-            if "__" not in key and hasattr(value, "get_params"):
-                estimators.append(value)
-        if len(estimators):
-            return _VisualBlock("parallel", estimators, names=None)
+        # Only look at the estimators in the first layer
+        estimators = [
+            (key, est)
+            for key, est in estimator.get_params().items()
+            if (
+                "__" not in key
+                and hasattr(est, "get_params")
+                and hasattr(est, "fit")
+                and callable(est.fit)
+            )
```

Comment:
Shouldn't get_params be called with deep=False ?

Reply:
I agree setting `deep=False` works here. I updated the PR with your suggestion.